tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/nvidia-imex-status.job

```diff
@@ -8,6 +8,6 @@
 sleep 45
 QUEUE_NAME=$(cat "/etc/chef/dna.json" | jq -r ".cluster.scheduler_queue_name")
 COMPUTE_RES_NAME=$(cat "/etc/chef/dna.json" | jq -r ".cluster.scheduler_compute_resource_name")
-IMEX_CONFIG_FILE="/opt/parallelcluster/shared/nvidia-imex/config_${QUEUE_NAME}_${COMPUTE_RES_NAME}.cfg"
+IMEX_CONFIG_FILE="/etc/nvidia-imex/config.cfg"
```

Comment:
We can remove this file. We should no longer specify the configuration file if not needed!

Reply:
I remember it was necessary, but apparently it is not when the default location is used